<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1368</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/bkg5.jpg);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:rgb(214, 214, 214)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="styleSheet">
       <string notr="true">border: none;
background: transparent;

</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>459</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>459</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"> background: transparent</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border:2px solid grey;
</string>
            </property>
            <property name="title">
             <string>      SignIn with your Account      </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <property name="topMargin">
                <number>6</number>
               </property>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="topMargin">
                <number>12</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="styleSheet">
                  <string notr="true">border: none;
background: transparent;</string>
                 </property>
                 <property name="text">
                  <string>Username</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_username"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="topMargin">
                <number>5</number>
               </property>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="styleSheet">
                  <string notr="true">border:none;
background: transparent;</string>
                 </property>
                 <property name="text">
                  <string>Password </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_password"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="topMargin">
                <number>38</number>
               </property>
               <item alignment="Qt::AlignLeft">
                <widget class="QPushButton" name="pushButton_Login">
                 <property name="styleSheet">
                  <string notr="true">border:none;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                 <property name="default">
                  <bool>false</bool>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 2px solid grey;
</string>
            </property>
            <property name="title">
             <string>Don't have an Account? SignUp</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <property name="spacing">
                <number>7</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>15</number>
               </property>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="styleSheet">
                  <string notr="true">border:none;
background: transparent;</string>
                 </property>
                 <property name="text">
                  <string>Username</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_userNamaSignUp"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="topMargin">
                <number>5</number>
               </property>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="styleSheet">
                  <string notr="true">border:none;
background: transparent;</string>
                 </property>
                 <property name="text">
                  <string>Password </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_passwordSignUp"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="styleSheet">
                  <string notr="true">border:none;
background: transparent;</string>
                 </property>
                 <property name="text">
                  <string>Confirm
Password </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_confPasswordSignUp"/>
               </item>
              </layout>
             </item>
             <item alignment="Qt::AlignLeft">
              <widget class="QPushButton" name="pushButton_signup">
               <property name="styleSheet">
                <string notr="true">border: transparent;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1368</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
